src/web/components/MostViewed/MostViewedFooter/MostViewedFooterData.tsx

```diff
@@ -50,13 +50,26 @@ function buildSectionUrl(
 	return joinUrl([ajaxUrl, `${endpoint}?dcr=true`]);
 }
 
+function buildDeeplyReadUrl(ajaxUrl: string, pillar: CAPIPillar) {
+	return joinUrl([ajaxUrl, 'most-read-deeply-read', `${pillar}.json?dcr`]);
+}
+
 export const MostViewedFooterData = ({
 	sectionName,
 	pillar,
 	ajaxUrl,
 	design,
 }: Props) => {
-	const url = buildSectionUrl(ajaxUrl, pillar, sectionName);
+	const ABTestAPI = useAB();
+
+	const inDeeplyReadTestVariant = ABTestAPI.isUserInVariant(
+		'DeeplyReadTest',
+		'variant',
+	);
+	// const url = buildSectionUrl(ajaxUrl, pillar, sectionName);
+	const url = inDeeplyReadTestVariant
+		? buildDeeplyReadUrl(ajaxUrl, pillar)
+		: buildSectionUrl(ajaxUrl, pillar, sectionName);
```

Comment:
This does read really nice so I'm hesitant if this works or not but ccould we maybe inline the joinUrl call? I guess it's a trade of of the abstraction meaning you have too move around the file to see what happening (plus you don't know if there's anything else using that code) vs. it being immediately clear what happening but not as pretty?

Reply:
I guess my main reasoning for abstracting this way is that building a section URL is a bigger process and when/if it comes to removing the test logic it should be much easier to understand which parts relate to deeply read and which do not. Ultimately they both use joinUrl but I thought the difference between them merited having them split into their own functions.

Reply:
> when/if it comes to removing the test logic it should be much easier to understand which parts relate to deeply read and which do not

👍 that's a really sound reason - great point